shared/src/main/scala/com/comcast/ip4s/MulticastSocketAddress.scala

```diff
@@ -100,6 +102,13 @@ object MulticastSocketAddress {
     }
   }
 
+  // Non-implicit because this conflicts with the Order instance
```

Comment:
@matthughes Do you remember why this is non-implicit? Why just this one and not the other eq instances?

Reply:
Looks like it was added in 0681ea88a883e03b6d3fce561b2bd3a2335e4218

Reply:
OK, I was able to remove all the explicit `Eq` instances, as they were all redundant with `Order` instances.